core/storage/trie/impl/polkadot_trie_db.cpp

```diff
@@ -80,6 +80,10 @@ namespace kagome::storage::trie {
     return outcome::success();
   }
 
+  void PolkadotTrieDb::recreateOnState(const common::Buffer &merkle_hash) {
```

Comment:
Hm... I guess it's bad name of method. This doing not the same what tell his name. Pls, comment or pick up more appropriate name.